packages/server-web/src/client/component/TodoItem.vue

```diff
@@ -71,7 +72,7 @@ export default {
       return this.$props.todo.trackedTimes || [];
     },
     todoTrackedTimesString() {
-      return this.todoTrackedTimes.map(tracked => "#TRACK(" + JSON.stringify(tracked) + ")");
+      return this.todoTrackedTimes.map(tracked => "#track(" + JSON.stringify(tracked) + ")");
```

Comment:
is it generally case insensitive?

Reply:
Yes, it's case insensitive and I hope we can keep it that way...